bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutationThrottling.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.grpc.async;
+
+import java.util.concurrent.TimeUnit;
+
+import com.google.cloud.bigtable.grpc.BigtableSessionSharedThreadPools;
+
+public class BulkMutationThrottling {
+
+  private static boolean isThrottling = false;
+
+  public static synchronized void throttle(final ResourceLimiter resourceLimiter,
```

Comment:
I don't love this being a static method, I feel like its something that the session should own